pkg/quarterdeck/api/v1/api.go

```diff
@@ -41,6 +41,7 @@ type QuarterdeckClient interface {
 
 	// Users Resource
 	UserUpdate(context.Context, *User) (*User, error)
+	UserRoleUpdate(context.Context, string, *UpdateRoleRequest) (*User, error)
```

Comment:
Little nit, If you're going to have a request structure, I prefer for the userID to be in that data structure, e.g.

```
UserRoleUpdate(context.Context, *UpdateRoleRequest) (*User, error)

type UpdateRoleRequest struct {
    ID ulid.ULID
    Role string
}
```

Reply:
I think that's definitely better.